src/ray/core_worker/transport/direct_task_transport.cc

```diff
@@ -51,10 +51,20 @@ Status CoreWorkerDirectTaskSubmitter::SubmitTask(TaskSpecification task_spec) {
               task_finisher_->CompletePendingTask(
                   task_id, push_task_reply, reply.actor_address());
             } else {
-              RAY_LOG(INFO) << "Failed to create actor " << actor_id
-                            << " with status: " << status.ToString();
+              rpc::RayErrorInfo ray_error_info;
+              if (status.IsSchedulingCancelled()) {
+                RAY_LOG(DEBUG) << "Actor creation cancelled, actor id = " << actor_id;
+                task_finisher_->MarkTaskCanceled(task_id);
+                ray_error_info.mutable_actor_died_error()->CopyFrom(reply.death_cause());
```

Comment:
Should we add an error message that the actor scheduling has cancelled? (maybe the current way is good enough?)

Reply:
Yeah I was looking at that. 🤔 The actor death cause provides the underlying method of cancellation (e.g. `ray.kill`), but I'll check to see what the error message looks like.

Reply:
Sgtm. I think the benefit of having better error message is it indicates if the actor was ever created or not. But I think it's okay not to handle it if it is too hard.